utility.go

```diff
@@ -0,0 +1,29 @@
+package steamworks
+
+import (
+	"runtime"
+	"unsafe"
+)
+
+const is32Bit = unsafe.Sizeof(int(0)) == 4
+
+func cStringToGoString(v uintptr, sizeHint int) string {
+	bs := make([]byte, 0, sizeHint)
+	for i := int32(0); ; i++ {
+		b := *(*byte)(unsafe.Pointer(v))
+		v += unsafe.Sizeof(byte(0))
+		if b == 0 {
+			break
+		}
+		bs = append(bs, b)
+	}
+	return string(bs)
+}
+
+// Helper function to convert Go string to C string
```

Comment:
Is this function needed? A function registered for PureGo can treat Go's string directly